README.md

```diff
@@ -31,80 +32,53 @@ jobs:
           reporter: github-pr-check # Change reporter. (Only `github-pr-check` is supported at the moment).
 ```
 
-See the Inputs section below for details on the defaults and optional configuration settings.
-
 ## Inputs
 
 ### `github_token`
 
 **Required**. Must be in form of `github_token: ${{ secrets.github_token }}`'.
 
-### workdir
-
-**Optional**. The directory to run remark-lint in. Default is `.`.
-
-### remark_flags
-
-**Optional**. Additional flags you want to pass to remark-lint. Default is ` `.
+### `workdir`
 
-### `level`
-
-**Optional**. Report level for reviewdog \[info, warning, error\].
-It's same as `-level` flag of reviewdog.
+**Optional**. The directory to run remark-lint in. Defaults to `.`.
 
-### `reporter`
-
-**Optional**. Reporter of reviewdog command \[github-pr-check, github-pr-review, github-check\].
-Default is github-pr-check. github-pr-review can use Markdown and add a link to rule page in reviewdog reports.
+### `format`
 
-**NB:** Only `github-pr-check` is supported currently.
-
-#### `filter_mode`
-
-**Optional**. Filtering mode for the reviewdog command \[added, diff_context, file, nofilter\]. Default = `"added"`.
+**Optional**. If true, remark-lint format files and commit are creatable (use other Action). Defaults to `false`.
 
 #### `fail_on_error`
 
-**Optional**. Exit code for reviewdog when errors are found \[`true`, `false`\]. Default = `false`.
-
-#### `reviewdog_flags`
+**Optional**. Exit code for when remark-lint errors are found \[`true`, `false`]. Defaults to `false`.
 
-**Optional**. Additional reviewdog flags. Default = `""`.
-
-#### `tool_name`
+#### `remark_flags`
 
-**Optional**. Tool name to use for reviewdog reporter. Default = `remark-lint`.
+**Optional**. Additional remark-lint flags. Defaults to `""`.
 
-## Development
+### `annotate`
 
-### Release
+**Optional**. Annotate remark-lint changes using reviewdog. Defaults to `true`.
 
-#### [haya14busa/action-bumpr](https://github.com/haya14busa/action-bumpr)
+#### `tool_name`
 
-You can bump version on merging Pull Requests with specific labels (bump:major,bump:minor,bump:patch).
-Pushing tag manually by yourself also work.
+**Optional**. Tool name to use for reviewdog reporter. Defaults to `remark-lint`.
 
-#### [haya14busa/action-update-semver](https://github.com/haya14busa/action-update-semver)
+### `level`
 
-This action updates major/minor release tags on a tag push. e.g. Update v1 and v1.2 tag when released v1.2.3.
-ref: <https://help.github.com/en/articles/about-actions#versioning-your-action>
+**Optional**. Report level for reviewdog \[info, warning, error]. It's same as `-level` flag of reviewdog. Defaults to `error`.
```

Comment:
```suggestion
**Optional**. Report level for reviewdog `\[info, warning, error]`. It's same as `-level` flag of reviewdog. Defaults to `error`.
```

Reply:
@limakzi Good catch! Thanks a lot for reading through the docs to find these syntax errors!